### PR TITLE
Adds the maven archetype for the creation of OrbisGIS wps script plugin.

### PR DESCRIPTION
**Pull request by SPalominos**
The maven archetype allow to add wps script to the WpsService.
The scripts are placed into srv/main/resources/scripts.
The icons are placed into srv/main/resources/icons.
The main class is WpsScriptsPackage.

### CODE REVIEW

File: wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+                }
+            } catch (IOException e) {
+                LoggerFactory.getLogger(WpsScriptsPackage.class).error(e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * This method loads the scripts one by one under different node path with different icons.
+     * (Be careful before any modification)
```
Comment (ebocher) [inline wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java]: But there is only one script ?

File: wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+ *      processFile : The File object corresponding to the script. Be careful, the plugin resource files can't be
+ *              accessed from the outside of the plugin. So you have to copy it (in a temporary file as example) before
+ *              adding it to the WpsService.
+ *      icon : Array of Icon object to use for the WpsClient tree containing the processes. The first icon will be used
+ *              for the first node of the path, the second icon for the second node ... If the node already exists,
+ *              its icon won't be changed. If there is less icon than node, the last icon will be used for the others.
+ *              If no icon are specified, the default one from the WpsClient will be used.
+ *      boolean : it SHOULD be true. Else the used will be able to remove the process from the WpsClient without
+ *              deactivating the plugin.
+ *      nodePath : Path to the node where the process should be add. If nodes of the path doesn't exists, they will be
```
Comment (ebocher) [inline wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java]: the node ?

File: wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java (hunk 1 above)
Comment (SPalominos) [inline wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java]: In the archetype, there is only one script (to keep it the simplest), but the plugin can contains as many script as wanted.
The 'defaultLoadScript()' method uses the 'recursiveScriptLoad()' method to load all the scripts keeping the file tree structure.
The 'customLoadScript' should load the script one by one. As there is only one script in the archetype, only one script is loaded. If the user put more scripts, he should duplicate the code loading the signle script.
The user should use one of these two methods.

File: wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java (hunk 2 above)
Comment (SPalominos) [inline wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java]: In the WpsClient, the processes are organized under a tree which contains nodes, subnodes, subsubnodes .... A process is a node, a folder is a node ... 
So in this archetype, the user can set under which nodes the process should be add.

**Pull request merged**